device-types/Cisco/C1000-8T-2G-L.yaml

```diff
@@ -7,43 +7,36 @@ comments: '[Cisco Catalyst 1000 Series Switches Data Sheet](https://www.cisco.co
 u_height: 1
 is_full_depth: false
 console-ports:
-  - name: Serial Console
+  - name: con0
     type: rj-45
   - name: USB con0
-    type: usb-mini-b
-  - name: USB con1
     type: usb-a
 power-ports:
-  - name: PS-1
+  - name: Built-in
```

Comment:
Current documentation addresses the PSUs as PSU1, PSU2, etc. Please revert this to PS-1 or PSU1 (my preference is PSU1), or provide documentation where it is called Build-in. Thanks!